### ISSUE TITLE
[Other - Accessibility] Collision between radio inputs and additional info components. (00.00.1)

### ISSUE THREAD

**Issue by chaseakirby** [department-of-veterans-affairs/va.gov-team#40370]
#### VFS team name
DGIB - My Education Benefits

#### VFS product name
My Education Benefits

#### VFS feature name
My Education Benefits App

#### Point of Contact/Reviewers
Brian DeConinck (@briandeconinck) - Accessibility

*For more information on how to interpret this ticket, please refer to the [Anatomy of a Staging Review issue ticket](https://depo-platform-documentation.scrollhelp.site/collaboration-cycle/Anatomy-of-a-Staging-Review-Issue-ticket.2060320997.html) guidance on Platform Website.

---

### Platform Issue
No VA.gov Experience Standard for the issue found.

### Issue Details
In Step 4 of 7: Benefit selection, users who are eligible for another benefit are prompted to choose a benefit that they must give up with an expandable additional info component about each benefit. As currently coded, this is presented roughly like this:

```
<input type="radio" id="some-id" ... />
<label for="some-id">
        Name of benefit
        <div class="form-expanding-group">... Learn more ...</div>
</label>
```

Since the "Learn more" additional info component is inside the `<label>`, expanding it also triggers a selection of the radio button associated with the label. It's possible that a user might accidentally make a selection when they only intended to learn more about that option --- and for some users likely won't realize what happened (eg. blind users, low vision users at high zoom). Since there is an opportunity to review at the end of the form this doesn't technically violate [WCAG 3.3.4](https://www.w3.org/TR/WCAG21/#error-prevention-legal-financial-data) but since this is a decision that can't be changed after submission we should be extra careful about preventing mistakes.

### Link, screenshot or steps to recreate
1. Navigate to the My Education Benefits app on staging and log in as test user 38.
2. Complete the form up to step 4 of 7.
3. When prompted to give up a benefit, select the first benefit radio button.
4. Click the "Learn more" that follows the second benefit radio button.
5. Note that the second radio button is now selected.

### VA.gov Experience Standard
[Category Number 00, Issue Number 00](https://depo-platform-documentation.scrollhelp.site/collaboration-cycle/VA.gov-experience-standards.1683980311.html)

### Other References

---

### Platform Recommendation
The "Learn more" additional info component should be moved outside of the `<label>` element and be associated with the `<label>` using aria-describedby. This may also require some extra styling and keyboard testing to make sure the tab order still makes sense, so please reach out in this issue or in Slack to discuss as needed!

Another alternative to consider: Combine all of the "Learn more" information across all of the available options into a single additional info component that appears between the question text and the radio options. That would resolve this issue and have a fairly simple keyboard pattern that's already in use elsewhere.

### VFS Team Tasks to Complete

- [ ] Comment on the ticket if there are questions or concerns
- [ ] Close the ticket when the issue has been resolved or validated by your Product Owner. If a team has additional questions or needs Platform help validating the issue, please comment in the ticket.

**Comment by chaseakirby** [comment]
@briandeconinck Hey Brian, can you update the ticket title with an accurate description but keep the "[Other]" and "(00.00.1)"?

**Comment by cesaresantiago** [comment]
We will address this on JIRA issue https://jira.np.afsp.io/browse/SA-17890

**Comment by cesaresantiago** [comment]
We will address this on JIRA issue https://jira.np.afsp.io/browse/SA-17890
